MMCore/MMCore.h

```diff
@@ -619,7 +619,7 @@ class CMMCore
    void setParentLabel(const char* deviceLabel,
          const char* parentHubLabel) throw (CMMError);
 
-   std::vector<std::string> getInstalledDevices(const char* hubLabel) throw (CMMError);
+   std::vector<std::string> getInstalledDevices(const char* hubLabel, bool force = false) throw (CMMError);
```

Comment:
i only put that in there a) to retain the ability to use the "old" behavior... because b) I wasn't 100% certain whether there was ever a valid use case for calling getInstalledDevices prior to device init.  I can remove it if you don't think so

Reply:
changed